src/libs/Navigation/AppNavigator/Navigators/RightModalNavigator.js

```diff
@@ -2,116 +2,95 @@ import React from 'react';
 import {createStackNavigator} from '@react-navigation/stack';
 
 import * as ModalStackNavigators from '../ModalStackNavigators';
-import defaultModalScreenOptions from '../defaultModalScreenOptions';
```

Comment:
the `defaultModalScreenOptions` is not used in the app anymore right? So we can remove it